### PR TITLE
Serve Model Azure Functions How-To

### PR DESCRIPTION
**Pull request by luisquintanilla**
Serve Model Azure Functions How-To

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net?branch=pr-en-us-12012)

#AB1484868

### CODE REVIEW

**Review by JRAlexander — APPROVED** [review]
LGTM!

File: docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
-# How-To: Use ML.NET Model in Azure Functions
+# How-To: Deploy ML.NET machine learning model to Azure Functions
 
-This how-to shows how individual predictions can be made using a pre-built ML.NET machine learning model through the internet in a serverless environment such as Azure Functions.
-
-> [!NOTE]
-> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
-
-This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning github repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes).
+Learn how to deploy a pre-trained ML.NET machine learning model for predictions over the internet through an Azure Functions serverless environment.
```
Comment (natke) [inline docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md]: Over the internet --> in the cloud?? maybe

**Comment by natke** [comment]
Make sure it is clear that these are alternatives

Pre-trained model.
    - Use the [ML.NET Sentiment Analysis tutorial](../tutorials/sentiment-analysis.md) to build your own model **or** (<-- or similar)
    - Download this [pre-trained sentiment analysis machine learning model]

**Comment by natke** [comment]
This is a real nit but can we indent the code samples to align with the numbered list. I think this would increase readability

**Review by natke — APPROVED** [review]
LGTM. A couple of minor comments

File: docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/serve-model-serverless-azure-functions-ml-net.md]: #resolved

**Comment by CESARDELATORRE** [comment]
This is wrong doing that on a static object because the PredictionEngine is not threadsafe.
_predictionEngine = _mlContext.Model.CreatePredictionEngine<SentimentData, SentimentPrediction>(_model);

**Comment by CESARDELATORRE** [comment]
You could probably use a static PredictionEnginePool because the pool is thread-safe. 
Similar to how we use it in ASP.NET Core but without using dependency injection.

But I would really prefer to use dependency injection in the Azure Function in a similar way it is explained in this blog post:

https://blog.wille-zone.de/post/azure-functions-dependency-injection/

Then, the implementation using PredictionEnginePool would be very similar to the one recommended in ASP.NET Core WebAPI or apps.

**Pull request merged**